app/js/index.js

```diff
@@ -106,12 +115,11 @@ function init() {
       return +d.properties[field];
     });
 
-  var topoURL = DATA_DIRECTORY + "us-states.topojson";
-  //var topoURL = DATA_DIRECTORY + "CAcounty.topojson";
-  d3.json(topoURL, function(topology) {
+  
+  d3.json(DEFAULT_TOPO, function(topology) {
```

Comment:
Yeah, I think they will have a drop down list of files of pre-defined paths. So we can just check when init is called or onChange of the drop down menu. It all depend on their implementation.